src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+tree.addEventListener('click', e => {
+  const list = e.target.firstElementChild;
```

Comment:
It is better to use 'e.target.closest' instead of 'e.target.firstElementChild' to get the closest UL element.

Reply:
> e.target.closest

no is not!
MDN: The closest() method of the element interface traverses the element and its `PARENTS`
i`m check child element, not parent or target, so `closest` not suitable. Plz, don't complicated